lms/djangoapps/verify_student/models.py

```diff
@@ -597,7 +599,12 @@ def get_initial_verification(cls, user):
         Return:
             SoftwareSecurePhotoVerification (object)
         """
-        init_verification = cls.objects.filter(user=user, status__in=["submitted", "approved"])
+        init_verification = cls.objects.filter(
+            Q(photo_id_key__isnull=True) | Q(photo_id_key=''),
+            user=user,
+            status__in=["submitted", "approved"],
+            photo_id_key__isnull=False
```

Comment:
You should remove the `isnull` line at the end, since you have it at the start of the filter.